lib/labelhash.hh

```diff
@@ -82,19 +82,18 @@ protected:
     uint32_t _tag_labels_spin_lock;
 
 public:
+    khmer::Hashtable * graph;
 
-    LabelHash( WordLength ksize, std::vector<HashIntoType>& tablesizes)
-        : khmer::Hashbits(ksize, tablesizes)
+    LabelHash(Hashtable * ht) : graph(ht)
```

Comment:
This should be marked `explicit` to keep the compiler from auto converting a `Hashtable` to a `LabelHash` in erroneous code.